CHANGELOG.md

```diff
@@ -43,7 +43,7 @@
 * [ENHANCEMENT] Distributor: optimize sending of requests to ingesters by reusing memory buffers for marshalling requests. #5195 #5389
 * [ENHANCEMENT] Querier: add experimental `-querier.minimize-ingester-requests` option to initially query only the minimum set of ingesters required to reach quorum. #5202 #5259 #5263
 * [ENHANCEMENT] Querier: improve error message when streaming chunks from ingesters to queriers and a query limit is reached. #5245
-* [ENHANCEMENT] Use new data structure for labels, to reduce memory consumption. #3555
+* [ENHANCEMENT] Use new data structure for labels, to reduce memory consumption. #3555, #5731
```

Comment:
[nit] We've a script parsing CHANGELOG (used during the release process) and I think doesn't parse the numbers correctly with a `,`
```suggestion
* [ENHANCEMENT] Use new data structure for labels, to reduce memory consumption. #3555  #5731
```